samples/nrf9160/http_update/src/update.c

```diff
@@ -13,20 +13,32 @@
 #include <modem/bsdlib.h>
 #include <modem/modem_key_mgmt.h>
 #include <net/fota_download.h>
-#include <dfu/mcuboot.h>
+#include <net/socket.h>
+#include <nrf_socket.h>
+#include <stdio.h>
 
-#define LED_PORT	DT_GPIO_LABEL(DT_ALIAS(led0), gpios)
+#define LED_PORT DT_GPIO_LABEL(DT_ALIAS(led0), gpios)
 #define TLS_SEC_TAG 42
 
-static const struct	device *gpiob;
-static struct		gpio_callback gpio_cb;
-static struct k_work	fota_work;
+static const struct device *gpiob;
+static struct gpio_callback gpio_cb;
+static struct k_work fota_work;
```

Comment:
Was the original alignment not put through clang? I don't see any recent changes in our `.clang-format`. I would think that the formatter would keep this consistent.

Reply:
Correct, I will do the clang-format now since its being moved anyway.